setup.py

```diff
@@ -28,6 +28,7 @@
             'ga4gh_server=ga4gh.cli:server_main',
             'ga2vcf=ga4gh.cli:ga2vcf_main',
             'ga2sam=ga4gh.cli:ga2sam_main',
+            'ga4gh_repo=ga4gh.cli.repo_main',
```

Comment:
Should be ``ga4gh_repo=ga4gh:cli.repo_main``. Nice to see the new CI checks caught this!